rotkehlchen/tests/fixtures/blockchain.py

```diff
@@ -108,12 +109,13 @@ def fixture_kusama_manager(
     Consider mocking task manager, related with issue #2057: do not let task
     manager schedule any tasks during tests by default.
     """
-    manager = SubstrateManager(
+    kusama_manager = SubstrateManager(
```

Comment:
@LefterisJP could you help me to make this fixture work in the `rotkehlchen_api_server` fixture? Point me where I have to add it.
Now I've moved the connection when the class is instantiated I can't get connection with the nodes, therefore [this test](https://github.com/rotki/rotki/pull/2030/commits/97f7f0805f0803cfab1bbaef5e8956cd060babda#diff-0a79432e001c52537afa5686061a1f9521348e7f29c23d0ee40474b1cd17ab1fR1622) (above) fails. Thanks!

Reply:
Answered in discord. You already initialize the substrate manager in user unlock so this fixture is not needed in rotkehlchen_api_server fixture.